src/figuresClasses.ts

```diff
@@ -1,11 +1,71 @@
-export interface Figure {}
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
+type GetArea = () => number;
 
-export class Triangle implements Figure {}
+const roundArea = (area: number): number => Math.floor(area * 100) / 100;
 
-export class Circle implements Figure {}
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea: GetArea;
+}
+
+export class Triangle implements Figure {
+  shape: Shape = 'triangle';
+
+  constructor(
+    public color: Color,
+    public a: number,
```

Comment:
Let's use more explicit naming here. The first var is color, so why second is just 'a'? 
Let's call it a 'sideA' and so on :)